packages/synapse-react-client/src/synapse-client/SynapseClient.ts

```diff
@@ -1383,6 +1389,79 @@ export const getUserChallenges = (
   return doGet(url, accessToken, BackendDestinationEnum.REPO_ENDPOINT)
 }
 
+/**
+ * Get a list of teams registered to the given challenge.
+ * see http://rest-docs.synapse.org/rest/GET/challenge.html
+ */
+export const getChallengeTeams = (
+  accessToken: string | undefined,
+  challengeId: string | number,
+  offset: string | number = 0,
+  limit: string | number = 200,
+): Promise<ChallengeTeamPagedResults> => {
+  const url = `/repo/v1/challenge/${challengeId}/challengeTeam?&offset=${offset}&limit=${limit}`
+  return doGet(url, accessToken, BackendDestinationEnum.REPO_ENDPOINT)
+}
+
+/**
+ * Get a list of all teams registered to the given challenge.
+ * see http://rest-docs.synapse.org/rest/GET/challenge.html
+ */
+export const getAllChallengeTeams = (
+  accessToken: string | undefined,
+  challengeId: string | number,
+  offset: string | number = 0,
+  limit: string | number = 200,
+): Promise<ChallengeTeam[]> => {
+  const url = `/repo/v1/challenge/${challengeId}/challengeTeam?&offset=${offset}&limit=${limit}`
+  const fn = () => {
```

Comment:
This function should take the limit and offset for getAllOfPaginatedService() to work properly

(limit: number, offset: number) => {
    const url = `/repo/v1/challenge/${challengeId}/challengeTeam?&offset=${offset}&limit=${limit}`

Reply:
Does not make sense for getAllChallengeTeams() to take an offset or limit

Reply:
Good catch

Reply:
I had unwittingly used another existing errant function as the basis for the structure of this one. Should be fixed now in both places.